helm: add action; on tetragon servicemonitor

HujinoKun opened the pull request:
Hello,

This PR adds the action:replace option to tetragon's monitor services in order to avoid an infinite loop on the ArgoCD with OutOfSync state

<!--
Thanks for contributing! Please ensure your pull request adheres to the following guidelines:

- [x] All commits contain a well written commit message and are signed-off (see [Submitting a pull request](https://tetragon.io/docs/contribution-guide/submitting-a-pull-request/)).
- [ ] All code is covered by unit and/or end-to-end tests where feasible.
- [ ] All generated files are updated if needed (see [Making changes](https://tetragon.io/docs/contribution-guide/making-changes/)).
- [ ] Provide a title or release-note blurb suitable for the release notes (see [guidelines](https://tetragon.io/docs/contribution-guide/release-notes/#release-note-blurb-in-pr)).
- [ ] Update documentation and write an upgrade note if needed (see [guidelines](https://tetragon.io/docs/contribution-guide/release-notes/#upgrade-notes)).
- [ ] Are you a user of Tetragon? Please add yourself to the [Users doc](https://github.com/cilium/cilium/blob/main/USERS.md) in the Cilium repository.
-->

Fixes : N/A

### Description
Currently, I have an infinite loop on my argocd instance when I deploy tetragon with argocd on the servicemonitor part.
![photo_2025-07-17_17-09-10](https://github.com/user-attachments/assets/c49c3473-2cb3-4163-90cd-908222d774de)

mtardy reviewed (approved):
Sure, it's already the default value so I see no problem specifying it explicitly.

HujinoKun commented on install/kubernetes/tetragon/Chart.yaml:
```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 1.4.0
+appVersion: 1.4.1
```
I've just reverted the modification :+1:

HujinoKun commented:
> By the way, you'll need to squash your commits because we rebase and merge. If you need to update the appVersion to make the CI happy, you can do that in a separate commit it makes sense.
@mtardy  I just squashed.

So now if I want the CI to be green, I have to make a new commit with the appversion bump?

I prefer to ask the question to avoid commits and back and forth ^^

HujinoKun commented:
> > > By the way, you'll need to squash your commits because we rebase and merge. If you need to update the appVersion to make the CI happy, you can do that in a separate commit it makes sense.
> > > @mtardy  I just squashed.
> > 
> > 
> > So now if I want the CI to be green, I have to make a new commit with the appversion bump?
> > I prefer to ask the question to avoid commits and back and forth ^^
> 
> I don't know, run `make -C install/kubernetes lint` locally to know or just `make -C install/kubernetes`

I've just added what the CI was missing.

Let me know if it's good from your side :)

michi-covalent reviewed (approved):
thank you! 🥰
